poll/poll.go

```
@@ -0,0 +1,163 @@
+// PollIncidents continuously monitors for incidents and updates the light status
+func PollIncidents(startTime time.Time, light lights.Light, logger *types.Logger) {
+	logger.InfoLog.Printf("Starting incident polling at %s", startTime.Format(time.RFC3339))
+	fmt.Printf("*** Starting incident polling at %s\n", startTime.Format(time.RFC3339))
+
+	notifiedIncidents := make(map[int]bool)
+	seenIncidents := make(map[int]bool)
+
+	for {
+		if err := network.CheckConnectivity(); err != nil {
+			logger.WarnLog.Printf("Internet connectivity issue: %s", err.Error())
+			light.On(lights.StateYellow)
```

golangci-lint (1.62.2)

Error return value of `light.On` is not checked

(errcheck)